src/licensedcode/data/rules/mit_or_gpl-2.0_68.RULE

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
Sorry to have missed this before, but why do we need to start a file with ---?

Reply:
This is beacause of the YAML forntmatter, and yes! We do this for the script all right, and the reindex fails if we don't give this:
```
Invalid rule files: '/home/ayansinha/nexB/write_access/scancode-extra/src/licensedcode/data/rules'
'While loading: file:///home/ayansinha/nexB/write_access/scancode-extra/src/licensedcode/data/rules/mit_or_gpl-2.0_68.RULE\nTraceback (most recent call last):\n  File "/home/ayansinha/nexB/write_access/scancode-extra/src/licensedcode/models.py", line 2123, in load_data\n    self.load(rule_file=rule_file)\n  File "/home/ayansinha/nexB/write_access/scancode-extra/src/licensedcode/models.py", line 2259, in load\n    raise e\n  File "/home/ayansinha/nexB/write_access/scancode-extra/src/licensedcode/models.py", line 2245, in load\n    raise InvalidRule(\nlicensedcode.models.InvalidRule: Cannot load rule with empty text: Rule(identifier=\'mit_or_gpl-2.0_68.RULE\', license_expression=None, minimum_coverage=0, is_continuous=False, relevance=100, has_stored_relevance=False, length=0): file:///home/ayansinha/nexB/write_access/scancode-extra/src/licensedcode/data/rules/mit_or_gpl-2.0_68.RULE\n'
```
so we should be good.